packages/asc-ui/src/utils/__tests__/themeUtils.test.ts

```diff
@@ -202,3 +204,20 @@ describe('fromTheme', () => {
     ).not.toThrow()
   })
 })
+
+describe('showAboveBackDrop', () => {
+  it('should set the correct z-index when there is a hasBackDrop prop or positive parameter', () => {
+    expect(showAboveBackDrop(true)({})[1]).toBeTruthy() // should return an array with the styling rules
+    expect(showAboveBackDrop(true)({})[1]).toContain(BACKDROP_Z_INDEX + 1)
+
+    expect(showAboveBackDrop(true)({ hasBackDrop: true })[1]).toBeTruthy() // should return an array with the styling rules
+    expect(showAboveBackDrop()({ hasBackDrop: true })[1]).toContain(
```

Comment:
perhaps add two more cases where parameter is true and hasBackDrop false. And vice-versa